internal/server/middlewares.go

```
@@ -113,10 +120,8 @@ func requestLogger(lg *zap.Logger) gorillamux.MiddlewareFunc {
 			case http.MethodGet:
 				break
 			default:
-				buf, err := io.ReadAll(req.Body)
-				if err != nil {
-					lg.Debug("error reading request body: %v", zap.String("error", err.Error()))
-				} else if len(buf) > 0 {
+				if noOfBytes > 0 {
+					buf, _ := io.ReadAll(bodyCopy)
```

should we not silently drop error here?
lets log the error?